app-ios/Sources/AnnouncementFeature/AnnouncementView.swift

```diff
@@ -0,0 +1,102 @@
+import appioscombined
+import ComposableArchitecture
+import Model
+import SwiftUI
+
+public struct AnnouncementState: Equatable {
+    public init(announcements: [AnnouncementsByDate] = []) {
+        self.announcements = announcements
+    }
+
+    public var announcements: [AnnouncementsByDate] = []
```

Comment:
This is not needed 👍🏼 
```suggestion
    public var announcements: [AnnouncementsByDate]
```

Reply:
I forgot to remove it..! Thanks to suggestion :)